maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/TestCases.java

```diff
@@ -0,0 +1,23 @@
+package aQute.bnd.maven.plugin;
+
+public enum TestCases {
+
+	junit3("${classes;EXTENDS;junit.framework.TestCase;CONCRETE}"),
+	junit4("${classes;HIERARCHY_ANNOTATED;org.junit.Test;CONCRETE}"),
+	junit5("${sort;${uniq;${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.jupiter.api.Test;CONCRETE},"
+		+ "${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.jupiter.api.RepeatedTest;CONCRETE},"
+		+ "${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.jupiter.params.ParameterizedTest;CONCRETE},"
+		+ "${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.jupiter.api.TestFactory;CONCRETE},"
+		+ "${classes;HIERARCHY_INDIRECTLY_ANNOTATED;org.junit.jupiter.api.TestTemplate;CONCRETE}}}"),
+	all("${sort;${uniq;" + junit3.filter() + ";" + junit4.filter() + ";" + junit5.filter() + "}}");
```

Comment:
You end up with 2 layers of `${sort;${uniq;` here. I would think that `${sort;${uniq;` should be done when setting the builder property.

Reply:
fixed